finality-aleph/src/new_network/mod.rs

```diff
@@ -8,6 +8,7 @@ use sp_runtime::traits::Block;
 use std::{borrow::Cow, collections::HashSet, pin::Pin};
 
 mod connection_manager;
+pub mod data_network;
```

Comment:
Rather than `pub`ing this whole thing, just publicly reexport the one thing you want from here (as far as I can tell this is `AlephNetworkData`?).

Reply:
Done

Reply:
Not really, this should be `mod data_network` and below `pub use data_network::DataForAleph` or whatever that thing is. Then you are importing it from here.